java/ql/src/Security/CWE/CWE-829/InsecureDependencyResolution.ql

```diff
@@ -0,0 +1,38 @@
+/**
+ * @name Failure to use HTTPS or SFTP URL in Maven artifact upload/download
+ * @description Non-HTTPS connections can be intercepted by third parties.
+ * @kind path-problem
```

Comment:
```suggestion
 * @kind problem
```
This should be `problem` instead of `path-problem`

Reply:
Yes.